### PR TITLE
Docs updated, Prerequisites section [20.3]

### PR DESCRIPTION
**Pull request by Karm**
SSIA
Tested also on on "clean-room" Ubuntu.

### CODE REVIEW

File: README-Mandrel.md [hunk 1]
```diff
@@ -35,16 +35,19 @@ Mandrel distributions can be downloaded from [the repository's releases](https:/
...
-dnf install glibc-devel zlib-devel gcc
+dnf install glibc-devel zlib-devel gcc freetype-devel libstdc++-static
 ```
 
+Note the package might be called `glibc-static` instead of `libstdc++-static`.
+
 On Ubuntu-like systems with:
 ```bash
-apt install libc6-dev zlib1g-dev gcc
+apt install gcc zlib1g-dev build-essential libfreetype6-dev
```
Comment (zakkak) [inline README-Mandrel.md:50]: isn't build-essential a bit aggressive/bloated?

Comment (Karm) [reply README-Mandrel.md:50]: @zakkak I had an issue with static glibc and this did the trick. I am by no means an Ubuntu user though, so feel free to suggest something different that would work on vanilla Ubuntu 18 and 20.

Comment (zakkak) [reply README-Mandrel.md:50]: Suggestion added in https://github.com/graalvm/mandrel-packaging/pull/96#discussion_r564465432 let's keep the discussion there and apply the changes everywhere later :)

**Comment by Karm** [comment]
@zakkak Updated. Ready to be merged.

File: README-Mandrel.md [hunk 2]
```diff
@@ -35,16 +35,19 @@ Mandrel distributions can be downloaded from [the repository's releases](https:/
 Mandrel's `native-image` depends on the following packages:
 * glibc-devel
 * zlib-devel
+* freetype-devel
 * gcc
```
Comment (zakkak) [inline README-Mandrel.md:39]: Shouldn't we include `libstdc++-static` in this list?

**Review by zakkak — APPROVED** [review]

File: README-Mandrel.md [hunk 3]
```diff
@@ -35,16 +35,19 @@ Mandrel distributions can be downloaded from [the repository's releases](https:/
...
 
 On Fedora/CentOS/RHEL they can be installed with:
 ```bash
-dnf install glibc-devel zlib-devel gcc
+dnf install glibc-devel zlib-devel gcc freetype-devel libstdc++-static
 ```
 
+Note the package might be called `glibc-static` instead of `libstdc++-static`.
```
Comment (zakkak) [inline README-Mandrel.md:46]: This note is missing from the other PRs

**Review by zakkak — CHANGES_REQUESTED** [review]
Thanks @Karm.
Please see the added comments.

**Pull request closed without merging**